### PR TITLE
Improve gostudy.py

### PR DESCRIPTION
**Pull request by emergenitro**
Hi, 

Changed gostudy.py slightly due to redundancy in the code and large use of if/else statements. This should make it slightly more efficient. Additionally, I apologise for the edits on app.py, I had the Black formatter on and it auto-formatted, and I noticed it just when I made this PR so just ignore app.py (it should be formatted better now though). Just double check gostudy.py

### CODE REVIEW

File: src/gostudy.py [hunk 1]
```diff
@@ -1,97 +1,74 @@
...
+async def gostudy(
+    interaction: discord.Interaction,
+    user: discord.User = discord.SlashOption(
+        name="name", description="Who do you want to use this command on? (only for Mods)", required=False
+    ),
+):
+    guild = bot.get_guild(GUILD_ID)
+    forced_mute_role = guild.get_role(FMROLE)
+
+    user_id = interaction.user.id if user is None else user.id
```
Comment (Sachin-dot-py) [inline src/gostudy.py:16]: In this updated `/gostudy` command code there's no isModerator check when choosing a particular user so anyone would be able to Study Mute another member.

**Review by Sachin-dot-py — CHANGES_REQUESTED** [review]
This pull request can be merged once the permissions issue in the `/gostudy` command is resolved.

**Comment by Sachin-dot-py** [comment]
Resolved the conflicts arising from merging PR #67

**Comment by Sachin-dot-py** [comment]
@emergenitro @juzcallmekaushik anyone able to make the requested changes?

**Comment by Juzcallmekaushik** [comment]
> ?

I am working on the recode rn.. I will add it into the AR24

**Comment by Sachin-dot-py** [comment]
Alright, closing this PR as per your comment.

**Pull request closed without merging**